desc/objectives/normalization.py

```diff
@@ -15,33 +15,34 @@ def compute_scaling_factors(thing):
     scales = {}
 
     def get_lowest_mode(basis, coeffs):
-        """Return lowest order coefficient (excluding m=0 modes)."""
+        """Return average (over n) of M=+/1 modes."""
```

Comment:
how is this the average? isn't it the max?

Reply:
mistype in the docstring, my bad